sys-schema/schema_unused_indexes.md

```diff
@@ -1,35 +1,11 @@
 ---
 title: sys Schema
```

Comment:
```suggestion
title: schema_unused_indexes
```

Reply:
@Oreoxmt  Maybe this should be `title: sys.schema_unused_indexes` instead?

Reply:
@dveeden To keep it consistent with other information_schema tables, we can use `title: schema_unused_indexes`
https://docs.pingcap.com/tidb/stable/information-schema-analyze-status

<img width="866" alt="image" src="https://github.com/pingcap/docs/assets/60599231/570176ee-1c43-4c82-a181-25c3c99dc6e9">